tests/unit/test_tokens.py

```diff
@@ -9,6 +9,51 @@ def test_client_token(aau_token_path):
     assert exists(aau_token_path)
 
 
+def test_client_token_no_nodeid(aau_token_path, mocker):
```

Comment:
tests the scenario where the C extensions for `uuid.getnode()` fail to return a result. In this case the client token should not be tied to a host id.